### PR TITLE
Added a blog post about the legacy branch of the TS SDK

### PR DESCRIPTION
**Pull request by JBBianchi**
**Many thanks for submitting your Pull Request :heart:!**

**What this PR does / why we need it**:
A new blog post about the legacy branch of the TS SDK and the associated package.

### CODE REVIEW

**Review by ricardozanini — APPROVED** [review]
👏

File: content/en/blog/releases/release-sdk-typescript-legacy.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+## Upcoming Packages with Correct Naming
+
+Moving forward, all new packages will be released under the correct (organization) name `@serverlessworkflow/sdk`. This change not only corrects a previous error but also aligns with the updated versioning of the spec. 
+
+The version 1.0.0-alpha of the package is still a [work in progress](https://github.com/serverlessworkflow/sdk-typescript/pull/188) and will be released once it is ready, featuring all the enhancements and updates introduced with the rewritten specification.
+
+
+## Why This Matters
+
+These changes ensure that we are consistent and aligned with our community's expectations and standards. By maintaining a legacy branch, we are providing a bridge for developers who may need more time to transition to the new specification. At the same time, the correct naming and versioning for new packages help streamline our offerings and reduce confusion.
```
[the remark this answers is not in the document]
Comment (JBBianchi) [reply content/en/blog/releases/release-sdk-typescript-legacy.md:42]: I'll integrate these changes but I think the original version puts the accent on the effort for the community with the `we`. Your suggestion is more concise and objective , but it feels a bit "colder".

File: content/en/blog/releases/release-sdk-typescript-legacy.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+npm install @severlessworkflow/sdk-typescript@4.0.0-rc1
+```
+
+This package will retain the older naming and organizational structure to maintain backward compatibility for existing projects.
+
+## Upcoming Packages with Correct Naming
+
+Moving forward, all new packages will be released under the correct (organization) name `@serverlessworkflow/sdk`. This change not only corrects a previous error but also aligns with the updated versioning of the spec. 
+
+The version 1.0.0-alpha of the package is still a [work in progress](https://github.com/serverlessworkflow/sdk-typescript/pull/188) and will be released once it is ready, featuring all the enhancements and updates introduced with the rewritten specification.
```
[the remark this answers is not in the document]
Comment (JBBianchi) [reply content/en/blog/releases/release-sdk-typescript-legacy.md:37]: I'll integrate the second half but for the first half, I think the emphasis in on the "version", I'll get rid of "package" altogether.

**Review by cdavernas — APPROVED** [review]
LGTM! Cheers!

On a side note, is CICD ready for publishing new posts totally our socials? If not, it might be a good idea to push that first, no?

**Comment by JBBianchi** [comment]
> On a side note, is CICD ready for publishing new posts totally our socials? If not, it might be a good idea to push that first, no?

This has only been been discussed in the meeting so far, I don't think any concrete action has been taken yet. I think we need to open a dedicated issue and assess the possible GH Action that exist to do the job and their requirements (such as API keys/tokens).

**Comment by cdavernas** [comment]
@ricardozanini would you be so kind to create said PR, so that we can start pushing new posts?

If no, no worries, I think can take care of it by the end of this week.

File: content/en/blog/releases/release-sdk-typescript-legacy.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ricardozanini) [reply content/en/blog/releases/release-sdk-typescript-legacy.md]: Yeah, I thought about that as well. I'm not hard opinionated on this. Feel free to ignore the comment regarding personal bias. 👍

**Comment by ricardozanini** [comment]
@cdavernas this week I might not be able to do it since I'm swamped with the token-exchange thing :P 

We can do it surely, do we have an issue?

**Review by ricardozanini — APPROVED** [review]
@JBBianchi, is there anything left, or can I merge it?

**Comment by cdavernas** [comment]
@ricardozanini no issue yet afaik, no

**Pull request merged**